s4e-web/src/app/views/settings/state/institution/institution.factory.spec.ts

```diff
@@ -4,8 +4,8 @@ import * as Factory from 'factory.ts';
 export const InstitutionFactory = Factory.makeFactory<Institution>({
   id: Factory.each(i => `institution:${i}`),
 
-  parentName : Factory.each(i => `parent #${i}`),
-  parentSlug: Factory.each(i => `parent-${i}`),
```

Comment:
Yes, but we must have parent which exists in the list, because now the InstitutionService is creating tree and filling 'ancestorDepth`. If you feed it a list of entities which have parentSlug which does not exist in the list it will break.